lua/entities/gmod_tardis/modules/sh_vortex.lua

```diff
@@ -98,6 +102,7 @@ else
 	})
 	
 	ENT:AddHook("Think","vortex",function(self)
+		if self:GetData("demat-fast",false) then return end
```

Comment:
This just done as a speed optimisation? It can go it's pretty optimised already if the vortex isn't enabled

Reply:
Sometimes the whole vortex fade-in messes with the reappear sequence and just pops the exterior into existence and breaking some things. This was an attempt to mitigate that but I don't think it has worked and the problem might be somewhere else.

Reply:
Okay if it's not helped can you remove this line then?